src/executorlib/executor/single.py

```
@@ -446,3 +448,17 @@ def create_single_node_executor(
+
+def _executor_self_test(exe):
+    f = exe.submit(sum, [1, 1])
+    counter = 0
+    while not f.done() and counter < 10:
+        sleep(0.1)
+        counter += 1
+    if not f.done():
+        exe.shutdown(wait=False, cancel_futures=False)
+        raise TimeoutError(
+            'Plase try "hostname_localhost=True" in the initialization of the SingleNodeExecutor(hostname_localhost=True).'
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)